libevmasm/SemanticInformation.cpp

```diff
@@ -29,6 +29,128 @@ using namespace std;
 using namespace solidity;
 using namespace solidity::evmasm;
 
+vector<SemanticInformation::Operation> SemanticInformation::readWriteOperations(Instruction _instruction)
```

Comment:
It's a lot easier to review this if this by making the switch more verbose, i.e., minimize the grouping (from `sstore` to  `log4`) and just return the vector corresponding to the instruction.  Because this code is critical in optimizations, I think the extra verbosity is okay.

Reply:
Are you saying it would be easier to review if we just had one return per case?

Reply:
Yeah, one return per case or fewer grouping.